guides/common/modules/con_troubleshooting.adoc

```diff
@@ -1,58 +1,6 @@
 [id="Troubleshooting_{context}"]
 = Troubleshooting
 
-ifdef::foreman-el,katello[]
-[id="troubleshooting_dnf_modules"]
-== DNF modules
-
-If the module fails to enable, it can mean an incorrect module is enabled. In that case, you have to resolve dependencies manually as follows. List the enabled modules:
-[options="nowrap" subs="+quotes,attributes"]
-----
-# dnf module list --enabled
-----
-
-=== Ruby
-
-If the Ruby 2.5 module has already been enabled, perform a module reset:
-[options="nowrap" subs="+quotes,attributes"]
-----
-# dnf module reset ruby
-----
-
-=== PostgreSQL
-
-If the PostgreSQL 10 module has already been enabled, perform a module reset:
-[options="nowrap" subs="+quotes,attributes"]
-----
-# dnf module reset postgresql
-----
-
-If a database was previously created using PostgreSQL 10, perform an upgrade:
-
-. Enable the DNF modules:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# dnf module enable {dnf-modules}
-----
-
-. Install the PostgreSQL upgrade package:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# dnf install postgresql-upgrade
-----
-
-+
-. Perform the upgrade:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# postgresql-setup --upgrade
-----
-endif::[]
-
-ifdef::katello[]
```

Comment:
This should not be dropped because these situations can only happen in a Katello installation.

Reply:
Addressed :+1: